umbraco/Umbraco-CMS
# Merging 7.15 - 8.1

A lot of manual merging with new features.

### Testing

* Make sure all tests pass (and JS ones)
* Make sure all pickers works correctly and adhere to the security settings of a user that has custom start nodes for both content and media. Create data types for each of these and do not select the option to bypass start nodes and test:
   * Media picker + Mini List View + Picker Search
   * Content picker + Mini List View + Picker Search
   * Multi url picker + Mini List View + Picker Search
   * RTE + Media picker + Mini List View + Picker Search
   * RTE + URL picker + Mini List View + Picker Search
   * Grid + RTE + Media picker + Mini List View + Picker Search
   * Grid + RTE + URL picker + Mini List View + Picker Search
* Then for each of the above data types, enable the bypass start nodes and test
* Check over the code for anything obvious
